src/EventHub/EventHub/help/Set-AzEventHubNamespace.md

```diff
@@ -1,28 +1,42 @@
 ﻿---
 external help file: Microsoft.Azure.PowerShell.Cmdlets.EventHub.dll-Help.xml
 Module Name: Az.EventHub
-online version: https://docs.microsoft.com/en-us/powershell/module/az.eventhub/set-azeventhubnamespace
+online version:
 schema: 2.0.0
 ---
 
 # Set-AzEventHubNamespace
 
 ## SYNOPSIS
-Updates the specified Event Hubs namespace.
+{{ Fill in the Synopsis }}
```

Comment:
Please regenerate the help document. `online version` and `SYNOPSIS` should not be empty.

Reply:
updated